ctlearn/data_loading.py

```diff
@@ -462,158 +494,182 @@ def get_image(self, run_number, event_number, tel_id):
         # Create image by indexing into the trace using the mapping table, then adding a
         # dimension to given shape (length,width,1)
         image = self._image_mapper.map_image(trace, tel_type)
+        image = np.array(image, dtype=np.float32)
```

Comment:
If this is intended just for dtype conversion (I think map_image already returns an array), it is probably preferred to use array.astype().

Reply:
Changed in bb8f905.